src/index.ts

```diff
@@ -54,6 +54,25 @@ export class TestxServer {
     return this.serverUrl;
   }
 
+  public async getDbSchema() {
```

Comment:
```suggestion
  public async getDatabaseSchema() {
```
I would personally prefer extensive names

Reply:
fixed